app/controllers/histories_controller.rb

```diff
@@ -8,9 +8,45 @@ class HistoriesController < ApplicationController
   def create
     redirect_to root_url unless current_user.can_create_subject_history?
 
-    History.create!(patient_id: params.permit(:patient_id)[:patient_id],
+    history = History.new(patient_id: params.permit(:patient_id)[:patient_id],
+                          created_by: current_user.email,
+                          comment: params.permit(:comment)[:comment],
+                          history_type: params.permit(:type)[:type] || 'Comment')
+
+    history.original_comment = history
+
+    # Attempt to save and continue; else if failed redirect to index
+    render(json: history.errors, status: 422) && return unless history.save
+
+    render(json: history) && return
+  end
+
+  # "Edits" a history comment - a new history comment is created with the updated comment text and a reference to the id of the original
+  def edit
```

Comment:
this is a little nit-picky, but I think it's more common to use the term `update` for an action like this (which is consistent with other controllers)

Reply:
I think because `update` is the default term, posting to `patient/:id/update` ends up calling `create` because it's a post.  I'd rather leave as is because of that

Reply:
in that case, I updated the http method to be consistent with the action: https://github.com/SaraAlert/SaraAlert/pull/793/commits/c8f1f62444e06ae8640e77e29be166045165d6dd#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5R69